src/Http/Requests/Account/UpdateUserPasswordRequest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Account;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateUserPasswordRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "password" => ["required", "string", "min:8", "current_password"],
+            "newPassword" => ["required", "string", "min:8", "confirmed"],
+            "newPassword_confirmation" => ["required", "string", "min:8"],
```

Comment:
We are using `camelSnake_case`?

Reply:
Please change this :)

Reply:
I'm afraid that's a Laravel convention.

Reply:
unfortunately it's laravel convention 
https://laravel.com/docs/9.x/validation#rule-confirmed

Reply:
but something like that `new_password` and `new_password_confirmation` will be better, I think.